packages/block-editor/src/components/rich-text/index.js

```diff
@@ -603,6 +607,11 @@ function RichTextWrapper(
 			fontFamily={ fontFamily }
 			fontWeight={ fontWeight }
 			fontStyle={ fontStyle }
+			minWidth={ minWidth }
+			maxWidth={ maxWidth }
+			onBlur={ onBlur }
+			setRef={ setRef }
+			id={ props.id }
```

Comment:
Hey @ellatrix 👋 I was wondering if you would have any idea about an issue I had, I added some more missing props for native, more specifically for the `Button` component that the `Buttons` block uses. If I destructure the `id` prop it breaks the web editor. Do you know by any chance why does it break?

If I pass the prop as `props.id` it works correctly. Any help would be appreciated, thanks!

Reply:
Hi! That's probably because, for web, props need to be spread on the container component itself (further down in this file) rather than passed through rich text.